content/providers/provider-azure/authentication.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Authentication 
+weight: 10
+description: Authentication options with the Upbound Azure official provider
+---
+
+The Upbound Official Azure Provider supports multiple authentication methods.
+
+* [Service principal with Kubernetes secret](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals?tabs=browser#service-principal-object)
+* [System-assigned managed identity](https://learn.microsoft.com/en-us/azure/aks/use-managed-identity#enable-managed-identities-on-an-existing-aks-cluster)
+* [User-assigned managed identity](https://learn.microsoft.com/en-us/azure/aks/use-managed-identity#bring-your-own-managed-identity)
+
+## Service principal with Kubernetes secret
+
+A service principal is an application within the Azure Active Directory that
+passes `client_id`, `client_secret`, and `tenant_id` authentication
+tokens to create and manage Azure resources. 
+
+### Create a service principal using the Azure CLI tool
+
```

Comment:
Add a `hint` with a  link to where they can get the Azure CLI . 

https://docs.microsoft.com/en-us/cli/azure/install-azure-cli